apim-migration-testing-tool-rest-assured/src/main/java/restapi/JsonReadWrite.java

```
@@ -0,0 +1,357 @@
+          Object obj = parser.parse(new FileReader(runtimeJsonPath));
+            jsonObject = (JSONObject) obj;
+            JSONArray apisList = (JSONArray)jsonObject.get("apis");
+            apisList.add(apiIdToJson);
+            jsonObject.put("apis", apisList);
+            try (FileWriter file = new FileWriter(runtimeJsonPath)) {
+              file.write(jsonObject.toJSONString()); 
+              file.flush();
+   
+          } catch (IOException e) {
+              e.printStackTrace();
+          }
+        } catch (Exception e) {
+          
+        }
+          
+   }
+    
+   public static String readApiId(int indexOfApiInList) {
+       
+       JSONObject jsonObject = new JSONObject();
+       JSONParser parser = new JSONParser();
+       try {
+           Object obj = parser.parse(new FileReader(runtimeJsonPath));
+           jsonObject = (JSONObject) obj;
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    
+       JSONArray apis = (JSONArray)jsonObject.get("apis");
+       JSONObject getApi = (JSONObject)apis.get(indexOfApiInList);
+       String getApiId = (String)getApi.get("apiId");
+       
+       return getApiId;
+   }
+    
+   public static void addAppToJson(String appId) {
+       
+       JSONObject jsonObject = new JSONObject();
+       
+       try {
+         JSONObject apiIdToJson = new JSONObject();
+         apiIdToJson.put("appId", appId);  
+         JSONParser parser = new JSONParser();
+         Object obj = parser.parse(new FileReader(runtimeJsonPath));
+           jsonObject = (JSONObject) obj;
+           JSONArray appsList = (JSONArray)jsonObject.get("apps");
+           appsList.add(apiIdToJson);
+           jsonObject.put("apps", appsList);
+           
+           try (FileWriter file = new FileWriter(runtimeJsonPath)) {
+             file.write(jsonObject.toJSONString()); 
+             file.flush();
+  
+         } catch (IOException e) {
+             e.printStackTrace();
```

Its bad practice to swallow the errors with printStackTrace. Shall we do proper error handling for this. Eg: Throwing a runtime exception. Logging the error properly.